extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -82,6 +93,7 @@ export class IframeTransport {
     } else {
       frameData = this.createCrossDomainIframe();
       this.ampWin_.document.body.appendChild(frameData.frame);
+      this.createLongTaskObserver_();
```

Comment:
Note: multiple creatives may use same amp-analytics 3p vendor. In this case, only the first one to load actually creates the iframe. It is (only) that one who does the Long Task observation.